src/subgraph.ts

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+let SUBGRAPH_URL = '';
+
+async function getCurrentExchangeRate() {
```

Comment:
Write proper TS. The function's return type is missing here, its missing for almost all functions in this file.

Reply:
done